cluster/ci/monitoring/mixins/prometheus/alertmanager.jsonnet

```diff
@@ -13,6 +13,7 @@ local routes = (import 'alertmanager.libsonnet').alertmanagerRoutes;
     group_wait: '30s',
     group_interval: '5m',
     repeat_interval: '2h',
+    continue: true,
```

Comment:
This is because https://prometheus.io/docs/alerting/configuration/#route

For example, an alert from 1 job failure to 2 slack channels.